env/master.sh

```diff
@@ -1,2 +1,2 @@
 export CBIOPORTAL_URL="http://www.cbioportal.org"
-export GENOME_NEXUS_URL="https://www.genomenexus.org"
+export GENOME_NEXUS_URL="https://genomenexus-pr-221.herokuapp.com"
```

Comment:
I guess this is a temporary change to make the pr work, let's not forget to change it back before merging.